react/features/app/functions.web.js

```diff
@@ -39,7 +39,8 @@ const _INTERCEPT_COMPONENT_RULES = [
      * @returns {UnsupportedMobileBrowser|void} If the rule is satisfied then
      * we should intercept existing component by UnsupportedMobileBrowser.
      */
-    () => {
+    // eslint-disable-next-line no-unused-vars
+    state => {
```

Comment:
why is this change needed? does flow complain?

Reply:
yes, flow first complains about incompatible type of the function and then eslint complains about unused vars. We were trying few approaches to fix and this one was the simplest.

Reply:
Ack!